packages/mdc-notched-outline/README.md

```diff
@@ -48,6 +48,16 @@ Mixin | Description
 --- | ---
 `mdc-notched-outline-color($color)` | Customizes the border color of the notched outlined.
 `mdc-notched-outline-idle-color($color)` | Customizes the border color of the idle outline.
+`mdc-notched-outline-stroke-width($width)` | Changes notched outline width to a specified pixel value.
+`mdc-notched-outline-corner-radius($radius)` | Sets the corner radius of the notched outline element to the given number.
+`mdc-notched-outline-idle-corner-radius($radius)` | Sets the corner radius of the notched outline element in idle state.
+> NOTE:
```

Comment:
Change this whole NOTE to

Because notched-outline has sibling elements, you need to call the "idle" Sass mixins with parent selectors.
Consider the following example HTML:
```html
<div class="foo-line-ripple-parent">
  <div class="mdc-notched-outline foo-line-ripple">
    <svg>
      <path class="mdc-notched-outline__path"/>
    </svg>
  </div>
  <div class="mdc-notched-outline__idle"></div>
</div>
```
In order to customize any "non-idle" part of notched-outline, use the `.foo-line-ripple` CSS selector:
```scss
.foo-line-ripple {
  @include mdc-notched-outline-color(fooColor);
}
```
But in order to customize any "idle" part of the notched-outline, you must use the `.foo-line-ripple-parent` CSS selector:
```scss
.foo-line-ripple-parent {
  @include mdc-notched-outline-idle-color(fooColor);
}
```